### ISSUE TITLE
Ubuntu 18.04 compilation error

### ISSUE THREAD

**Issue by muirdok** [Aorimn/dislocker#156]
For 
Linux mTPdP50 4.15.0-33-generic #36-Ubuntu SMP Wed Aug 15 16:00:05 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
Got error:

> $ ruby -v
> ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]
> $ make
> [  2%] Building C object src/CMakeFiles/dislocker.dir/dislocker.c.o
> In file included from /usr/include/ruby-2.5.0/ruby.h:33:0,
>                  from /home/kek/src/dislocker/include/dislocker/ruby.h:35,
>                  from /home/kek/src/dislocker/include/dislocker/accesses/accesses.h:38,
>                  from /home/kek/src/dislocker/src/dislocker.c:36:
> /usr/include/ruby-2.5.0/ruby/ruby.h:24:10: fatal error: ruby/config.h: No such file or directory
>  #include "ruby/config.h"
>           ^~~~~~~~~~~~~~~
> compilation terminated.
> src/CMakeFiles/dislocker.dir/build.make:62: recipe for target 'src/CMakeFiles/dislocker.dir/dislocker.c.o' failed
> make[2]: *** [src/CMakeFiles/dislocker.dir/dislocker.c.o] Error 1
> CMakeFiles/Makefile2:200: recipe for target 'src/CMakeFiles/dislocker.dir/all' failed
> make[1]: *** [src/CMakeFiles/dislocker.dir/all] Error 2
> Makefile:129: recipe for target 'all' failed
> make: *** [all] Error 2

**Comment by WhiteHusky** [comment]
Seems like ruby changed where the config is stored. Can be fixed with:
```bash
ln -s /usr/include/x86_64-linux-gnu/ruby-2.5.0/ruby/config.h config.h```

**Comment by paulstelian97** [comment]
Any chance the fix will actually end up in Release any soon?

**Comment by Aorimn** [comment]
A new version of dislocker has been released, v0.7.3 and should fix this issue.

Nice idea @ZaneCEO!
